src/Uno.UWP/Storage/Helpers/AssetsManager.wasm.cs

```diff
@@ -24,7 +24,7 @@ namespace Windows.Storage.Helpers
 {
 	internal partial class AssetsManager
 	{
-		private static readonly Lazy<Task<HashSet<string>>> _assets = new Lazy<Task<HashSet<string>>>(() => GetAssets(CancellationToken.None));
+		internal static readonly Lazy<Task<HashSet<string>>> _assets = new Lazy<Task<HashSet<string>>>(() => GetAssets(CancellationToken.None));
```

Comment:
```suggestion
		internal static Lazy<Task<HashSet<string>>> Assets { get; } = new Lazy<Task<HashSet<string>>>(() => GetAssets(CancellationToken.None));
```

and rename usages of `_assets` to `Assets`